### ISSUE TITLE
Explore alternative way to display hero image title

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#1079]
Currently the title for the hero image is displayed as an H1, this works when the title is hidden and when there is only one hero image on a page. However, if there are multiple hero images there are multiple H1s, which is a violation. Alternatively, if units do not want a title on their hero image and also want to hide the page title, this causes a violation where there is no H1 on the page.

I propose we change the heading tag on the hero image to either an H2 or P tag and explore ways to visually hide the page title instead of removing it completely.

**Comment by timo54** [comment]
I want to add to Maggie's comment. 

1. H1 tags are reserved for (1) highlighting the owner of the site (typically the department), and (2) announcing the page's primary  title. The recommended practice of up-to two H1s but no more is intended for the University, however, some automated tools will flag this as a violation because their ruleset is geared towards commercial sites where two H1 tags will be confusing. 
2. The H1 should not contain text that's not found in the page's HTML TITLE tag. The H1 should be a subset of the TITLE tag so that it helps orient blind users to the site they are on. The TITLE tag is the first thing announced when the user opens a web page, the H1 needs to be in agreement with this to help the user verify which site they are on. 
3. H-tags should NEVER be hidden. They are meant to visually convey the content structure of the page and will fail to do this if they are hidden. Often structural components like Landmarks with aria-label can be used for navigation to their respective locations  rather than hiding a H-tag. The Breadcrumb section or side-bar are good examples.

**Comment by trubach** [comment]
A partial solution to this is that most of the pages that use the Hero paragraph are on homepages. For the site's homepage, we can set the h1 to the page's title in the header. See the [Graduate College homepage](https://grad.illinois.edu/) as an example:

<img width="1177" height="658" alt="Image" src="https://github.com/user-attachments/assets/6c1ad230-3dfd-447f-ba93-5c2c37de96c0" />

This can't be done for interior landing pages, so we'll need to continue finding a solution for those.
